app/models/region_type.rb

```diff
@@ -0,0 +1,14 @@
+class RegionType < ApplicationRecord
```

Comment:
Definitely prefer `RegionType` to `RegionKind`, but wasn't there some concern about using `Type` in a model name?

Reply:
`type` as a field name is problematic, but model should be fine.